extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -279,7 +279,14 @@ private String buildRedirectUri(RoutingContext context, URI absoluteUri, String
                 .toString();
     }
 
-    private Cookie removeCookie(RoutingContext context, String cookieName) {
-        return context.response().removeCookie(cookieName, true);
+    private void removeCookie(RoutingContext context, TenantConfigContext configContext, String cookieName) {
+        ServerCookie cookie = (ServerCookie) context.cookieMap().get(cookieName);
+        if (cookie != null) {
+            cookie.setMaxAge(0);
```

Comment:
If you want the cookie to be invalidated, why don't you also reset the value?

Reply:
Because to clear a cookie you set max age 0. The value has no effect in this case.

Reply:
yes, I know. Just a bit useless to send the value over the wire IMHO.

Reply:
Hi @gsmet @stuartwdouglas yes, I've seen it being suggested, for the moment I've just reused the code from Vertx and added a path, but I suppose I can set its value as empty. @lodrantl mentioned it is less obvious the cookie has been invalidated when tracing the network requests if the name remains